$MirOS: ports/www/firesomething/patches/patch-security_coreconf_OpenBSD_mk,v 1.2.2.2 2008/12/02 19:40:51 bsiegert Exp $
$OpenBSD: patch-security_coreconf_OpenBSD_mk,v 1.3 2005/04/09 15:43:47 wilfried Exp $
--- security/coreconf/OpenBSD.mk.orig	Thu Nov 22 05:39:33 2007
+++ security/coreconf/OpenBSD.mk	Mon Nov 24 08:41:31 2008
@@ -56,7 +56,7 @@ OS_LIBS			+= -pthread
 DSO_LDOPTS		+= -pthread
 endif
 
-DLL_SUFFIX		= so.1.0
+DLL_SUFFIX		= so.${SO_VERSION}
 
 OS_CFLAGS		= $(DSO_CFLAGS) $(OS_REL_CFLAGS) -ansi -Wall -Wno-switch -pipe -DOPENBSD
 
@@ -65,7 +65,7 @@ OS_LIBS			= 
 ARCH			= openbsd
 
 DSO_CFLAGS		= -fPIC -DPIC
-DSO_LDOPTS		= -shared -fPIC -Wl,-soname,lib$(LIBRARY_NAME)$(LIBRARY_VERSION).$(DLL_SUFFIX)
+DSO_LDOPTS		= -shared -fPIC -Wl,-soname,lib$(LIBRARY_NAME)$(LIBRARY_VERSION).$(DLL_SUFFIX) -Bsymbolic
 DSO_LDFLAGS		=
 
 MKSHLIB			= $(CC) $(DSO_LDOPTS)
